src/main/flight/pid_luxfloat.c

```diff
@@ -80,22 +84,24 @@ STATIC_UNIT_TESTED int16_t pidLuxFloatCore(int axis, const pidProfile_t *pidProf
 
     SET_PID_LUX_FLOAT_CORE_LOCALS(axis);
 
+    pidInitFilters(pidProfile);
```

Comment:
It would be much better to separate PID init from PID runtime ...
Parameter change is not handled anyway